css/save-animation.css

```diff
@@ -0,0 +1,34 @@
+.eu-cookie-compliance-save-preferences-button {
+    position: relative;
+}
+
+.eu-cookie-compliance-save-preferences-button.spinning {
+    padding-right: 40px;
+}
+
+.eu-cookie-compliance-save-preferences-button.spinning:before {
```

Comment:
Can we use double colons for pseudo elements - `::before` to follow Drupal's coding standards.